lib/SILGen/SILGen.cpp

```diff
@@ -1835,7 +1837,7 @@ class SILGenModuleRAII {
     }
   }
 
-  SILGenModuleRAII(SILModule &M, ModuleDecl *SM) : SGM{M, SM} {}
+  SILGenModuleRAII(SILModule &M) : SGM{M, M.getSwiftModule()} {}
```

Comment:
`explicit` here - to combat C++ being *bad*

Reply:
Good catch!